### PR TITLE
Sample-App chapter 4

### PR DESCRIPTION
**Pull request by anhdtp-1773**
Create a Sample App

### CODE REVIEW

**Comment by phuonganhdt-1773** [comment]
Ready

**Comment by phuonganhdt-1773** [comment]
ready

**Comment by phuonganhdt-1773** [comment]
ready

**Comment by anhdtp-1773** [comment]
ready

**Comment by anhdtp-1773** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+  def full_title(page_title = "")
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (huuphuc93) [inline app/helpers/application_helper.rb]: I18n

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
-    <%= stylesheet_link_tag    'application', media: 'all',
-                                              'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    "application", media: "all",
```
Comment (huuphuc93) [inline app/views/layouts/application.html.erb]:
```suggestion
    <%= stylesheet_link_tag "application", media: "all",
```

File: test/controllers/static_pages_controller_test.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-require 'test_helper'
+require "test_helper"
```
Comment (huuphuc93) [inline test/controllers/static_pages_controller_test.rb]: xóa cached folder test bằng lệnh git rm --cached test/*

**Comment by huuphuc93** [comment]
commented

**Comment by phuonganhdt-1773** [comment]
ready

**Comment by anhdtp-1773** [comment]
ready

**Comment by huuphuc93** [comment]
commented
gộp commit thành 1 commit rồi merged nhé

**Pull request merged**